src/item.cpp

```diff
@@ -760,6 +760,74 @@ std::string item::info( bool showtext, std::vector<iteminfo> &iteminfo, int batc
     return info( iteminfo, showtext ? &iteminfo_query::all : &iteminfo_query::notext, batch );
 }
 
+// Generates a long-form description of the freshness of the given rottable food item.
+// NB: Doesn't check for non-rottable!
+const std::string get_freshness_description( const item *food_item )
```

Comment:
Why return a const object?

`food_item` should be a reference.

Reply:
> Why return a const object?

...habit? I'll fix it -- and the food_item. Give me a few minutes.